### DISCUSSION TITLE
Recent documents in the start menu

### DISCUSSION THREAD

**Discussion by oliver124810** [wixtoolset/issues#6923] [Questions] [answered]
I have successfully added a shortcut to the start menu for my application, using something similar to [this](https://wixtoolset.org/documentation/manual/v3/howtos/files_and_registry/create_start_menu_shortcut.html). Now I would like to have in the start menu the list of recent documents opened next to the application, like for most of the Windows applications in the start menu, but I can not find any information. Any hint would be appreciated.

**Comment by robmen** [comment, marked as answer]
AFAIK, recent documents are managed by Windows as the user opens documents. It isn't something you install.

**Comment by oliver124810** [comment]
I still do not get how the list is managed by Windows, but it does not seem to be a WiX topic indeed. Thanks!
